ios/FluentUI/Button/ButtonTokenSet.swift

```diff
@@ -181,40 +204,49 @@ public class ButtonTokenSet: ControlTokenSet<ButtonTokenSet.Tokens> {
             case .foregroundColor:
                 return .uiColor {
                     switch style() {
-                    case .accent:
+                    case .accent, .floatingAccent:
                         return theme.color(.foregroundOnColor)
                     case .outline, .subtle:
                         return theme.color(.brandForeground1)
                     case .danger:
                         return theme.color(.foregroundLightStatic)
                     case .dangerOutline, .dangerSubtle:
                         return theme.color(.dangerForeground2)
+                    case .floatingSubtle:
+                        return theme.color(.foreground2)
                     }
                 }
             case .foregroundDisabledColor:
                 return .uiColor { theme.color(.foregroundDisabled1) }
             case .foregroundPressedColor:
                 return .uiColor {
                     switch style() {
-                    case .accent:
+                    case .accent, .floatingAccent:
                         return theme.color(.foregroundOnColor)
                     case .outline, .subtle:
                         return theme.color(.brandForeground1Pressed)
                     case .danger:
                         return theme.color(.foregroundLightStatic)
                     case .dangerOutline, .dangerSubtle:
                         return theme.color(.dangerForeground2)
+                    case .floatingSubtle:
+                        return theme.color(.foreground2)
                     }
                 }
             case .titleFont:
                 return .uiFont {
                     switch size() {
                     case .large:
-                        return theme.typography(.body1Strong)
+                        return theme.typography(.body1Strong, adjustsForContentSizeCategory: !style().isFloating)
                     case .medium, .small:
-                        return theme.typography(.caption1Strong)
+                        return style().isFloating ? theme.typography(.body2Strong, adjustsForContentSizeCategory: !style().isFloating) : theme.typography(.caption1Strong, adjustsForContentSizeCategory: !style().isFloating)
```

Comment:
Since you're already checking `style().isFloating` at the start of the `return` statement, I think you're good to just pass `true` (or leave it out), and `false`.

Reply:
Oh right, nice, thanks!